cypress/integration/views/stakeholdergroups.view.ts

```diff
@@ -0,0 +1,15 @@
+// Widgets on Stakeholder groups tab view
+export const createNewStakeholdergroupButton = "button[aria-label=create-stakeholder-group]";
+export const stakeholdergroupTable = 'table[aria-label="App table"]';
```

Comment:
for such cases using double quotes (") as outer quote is preferable instead of single quote

Reply:
yeah. I will update it either way(single quote in double)